pcsx2-qt/Settings/ControllerGlobalSettingsWidget.cpp

```diff
@@ -144,6 +144,9 @@ ControllerLEDSettingsDialog::ControllerLEDSettingsDialog(QWidget* parent, Contro
 	linkButton(m_ui.SDL2LED, 2);
 	linkButton(m_ui.SDL3LED, 3);
 
+	SettingsInterface* sif = dialog->getProfileSettingsInterface();
+
+	SettingWidgetBinder::BindWidgetToBoolSetting(sif, m_ui.enableSDLPS5PlayerLED, "InputSources", "SDLPS5PlayerLED", false);
```

Comment:
This probably should be `ControllerSettingWidgetBinder::BindWidgetToInputProfileBool()`.

You could also disable the option when editing input profiles.

Reply:
> You could also disable the option when editing input profiles
Sorry I'm not understanding this part (not familiar with PCSX2 code)
I used `SettingWidgetBinder::BindWidgetToBoolSetting` to match the enhanced mode toggle.

I made the change and rebased, but not clear on the benefit/change result.

When doing this change it only works for 'Shared' profile. Changing the input profiles breaks the enable/disable behavior.

Reply:
Ok actually, its broken even before the change, where a new profile would have a 3-state checkbox.
Will look into this a bit more.

Reply:
Profiles _should_ trigger a reload of input sources, if that's not happening, then that's not your bug (but probably mine).

Reply:
... Disregard my prior messages (deleted). I now understand how the Profile works. It won't actively affect the controller until a game is using the Profile. It works as you suggested. Pushed and squashed. Thank you.

Reply:
> Profiles _should_ trigger a reload of input sources, if that's not happening, then that's not your bug (but probably mine).

Just to clarify:
Unchecking the Enhanced or my new LED Checkbox reloads the input sources.
If I switch to a new profile, anything changed there does *not* reload the inputs sources as no title is actively using the new profile. If I launch a title/game bound to the new profile the settings are then applied to the controller.

Reply:
Then that's probably working as intended. There is no "switching" of profiles, changing the dropdown only changes what profile is being **edited** (that's why the label is "editing profile", or at least I'm pretty sure I changed it to say that..)

Reply:
In that case marking as resolved as I've incorporated your suggestion.